### DISCUSSION TITLE
Welcome to measurement-strategy Discussions!

### DISCUSSION THREAD

**Discussion by scollis** [CROCUS-Urban/measurement-strategy#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to coordinate the CROCUS Measurement Strategy Team (MST):
This will be used to **publically** track certain projects and reach certaindecisionss. 

  To get started, check in below with an introduction of yourself and tell us about what you do with this community.

**Comment by jrobrien91** [comment]
Hello! My name is Joe O'Brien and I am an Atmospheric Science Software Developer at Argonne National Laboratory within Scott's Geospatial Computing Innovations and Sensing (GCIS) Department. My primary work will be assisting in the operation of the Argonne Testbed for Multi-scale Observational Science (ATMOS) site.
